packages/react-components/src/popover/src/PopoverTrigger.tsx

```diff
@@ -92,14 +102,14 @@ export function InnerPopoverTrigger({
     children,
     forwardedRef,
     ...rest
-}) {
+}: InnerPopoverTriggerProps) {
     const { isOpen, setIsOpen, triggerProps, overlayProps, arrowProps } = usePopup("dialog", {
         id,
         open,
         defaultOpen,
         onOpenChange,
         hideOnEscape: !persistent,
-        hideOLeave: !persistent,
```

Comment:
Let's go TypeScript!

Reply:
haha, oh well we need to test the behavior lol

This is all great that TS find this but it really showcase the lack of interaction tests and the huge debt which comes with this release :(((((